sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -78,6 +78,7 @@ void _auto_configure_addr(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
          *    locked here) */
         netif->ipv6.addrs_flags[idx] &= ~GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_MASK;
         netif->ipv6.addrs_flags[idx] |= GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_VALID;
+        _nib_bus_post(netif, GNRC_IPV6_NIB_EVENT_ADDR_VALID, &netif->ipv6.addrs[idx]);
```

Comment:
You forgot to post for when the [address becomes VALID in classic DAD](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c#L212-L213).

Reply:
fixed